openqabot/types/baseconf.py

```diff
@@ -23,7 +23,8 @@ def __call__(
     ) -> List[Dict[str, Any]]:
         pass
 
-    @abstractstaticmethod
```

Comment:
why this ?

Reply:
https://docs.python.org/3/library/abc.html#abc.abstractclassmethod